my-app/pages/index.js

```diff
@@ -261,6 +260,12 @@ export default function Home() {
     }
   }, [walletConnected]);
 
+  const handelInputChange = (e) => {
```

Comment:
can you fix the typo here please, will merge

Reply:
fixed